website/data/cloud-docs-nav-data.json

```diff
@@ -37,6 +37,9 @@
         "path": "api-docs/cost-estimates"
       },
       { "title": "Feature Sets", "path": "api-docs/feature-sets" },
+      { "title": "GitHub App", "routes": [
+        { "title": "Installations", "path": "api-docs/github-app/installations" }
```

Comment:
Why have this be a child of `GitHub App` and not the base page? Is the plan for `Installations` to have sisters? If so it might make sense to add an overview page for this section.

Reply:
Good point, at the moment we only plan on having installations be surfaced.

We will hoist and have it be `GitHub App Installations`